integration/build/build_session_test.go

```diff
@@ -40,8 +33,9 @@ func TestBuildWithSession(t *testing.T) {
 	)
 	defer fctx.Close()
 
-	out := testBuildWithSession(t, client, client.DaemonHost(), fctx.Dir, dockerfile)
-	assert.Check(t, is.Contains(out, "some content"))
+	res, err := buildutil.Build(client, buildutil.BuildInput{ContextDir: fctx.Dir, Dockerfile: []byte(dockerfile)}, types.ImageBuildOptions{})
```

Comment:
This is failing (at least locally for me); looks like it fails to initialise a session;

```
--- FAIL: TestBuildWithSession (3.06s)
    daemon.go:296: [d96eb6cff800c] waiting for daemon to start
    daemon.go:328: [d96eb6cff800c] daemon started
    build_session_test.go:37: assertion failed: error is not nil: ContextDir not implemented without session
    daemon.go:283: [d96eb6cff800c] exiting daemon
```

Reply:
Looks that it happened after rebuild too